### PR TITLE
Remove fomantic input module

### PR DESCRIPTION
**Pull request by silverwind**
Another pure CSS module. Some styling is part of the `form` module which will likely follow next.

### CODE REVIEW

**Review by delvh — APPROVED** [review]

**Comment by silverwind** [comment]
Loading state also fixed:

Screenshot 2024-03-31 at 00 32 46

File: web_src/js/components/DashboardRepoList.vue [hunk 1]
```diff
@@ -355,9 +355,9 @@ export default sfc; // activate the IDE's Vue plugin
         </a>
       </h4>
       <div class="ui attached segment repos-search">
-        <div class="ui small fluid action left icon input" :class="{loading: isLoading}">
+        <div class="ui small fluid action left icon input">
           <input type="search" spellcheck="false" maxlength="255" @input="changeReposFilter(reposFilter)" v-model="searchQuery" ref="search" @keydown="reposFilterKeyControl" :placeholder="textSearchRepos">
-          <i class="icon"><svg-icon name="octicon-search" :size="16"/></i>
+          <i class="icon" :class="{'is-loading': isLoading}"><svg-icon name="octicon-search" :size="16"/></i>
```
Comment (wxiaoguang) [inline web_src/js/components/DashboardRepoList.vue]: It could also add a `small-loading-icon` to make the icon thinner.

**Review by denyskon — APPROVED** [review]

File: web_src/js/components/DashboardRepoList.vue (hunk 1 above)
Comment (silverwind) [reply web_src/js/components/DashboardRepoList.vue]: Looks almost too thin, but ok.

image

Comment (silverwind) [reply web_src/js/components/DashboardRepoList.vue]: Okay, in https://github.com/go-gitea/gitea/pull/30194/commits/6b96487b6e0d01d31e897834fbaa699157e8bd19 I renamed this class to `is-loading-2px` and added a `is-loading-3px` used here:

Screenshot 2024-03-31 at 14 39 33

Comment (silverwind) [reply web_src/js/components/DashboardRepoList.vue]: @wxiaoguang looking good?

Comment (wxiaoguang) [reply web_src/js/components/DashboardRepoList.vue]: TBH it looks strange. Just like writing something `tw-border tw-border-1`, it is counter-intuitive and seems wordy. Personally I prefer a clear name: `class="is-loading loading-icon-small` (loading-icon-medium)

Comment (silverwind) [reply web_src/js/components/DashboardRepoList.vue]: I think it's better to prefix with the loading class name as it's only meant to be used together.

Comment (silverwind) [reply web_src/js/components/DashboardRepoList.vue]: Renamed.

**Review by wxiaoguang — APPROVED** [review]

**Pull request merged**